requirements/ci-wheel.txt

```diff
@@ -15,11 +15,11 @@ tox==3.13.2
 trustme==0.5.2
 cryptography==2.7
 twine==1.13.0
+typing_extensions==3.7.4
```

Comment:
I'd put this into a separate PR. We'd also be able to backport it while figuring out CI stuff.

Reply:
Already done, already landed. :-)